local-dev.yml

```diff
@@ -215,6 +217,21 @@ services:
     depends_on:
       - hydroshare
       - companion
+  micro-auth:
+    image: scootna/micro-auth:latest
```

Comment:
shall we push this up to the hydroshare repository instead of `scootna`?

Reply:
definitely